docs/source/usage.rst

```diff
@@ -0,0 +1,185 @@
+                --make-html \
+                --html-output=./apidocs \
+                --project-base-dir "$(pwd)" \
+                --docformat=restructuredtext 
+
+        - name: Publish pydoctor documentation to the gh-pages branch
+          uses: peaceiris/actions-gh-pages@v3
+          with:
+            github_token: ${{ secrets.GITHUB_TOKEN }}
+            publish_dir: ./apidocs
+            commit_message: "Generate pydoctor documentation"
+
+.. note:: As mentionned in the ``actions-gh-pages`` documentation, the first workflow run won't actually publish the documentation to Github pages. 
+    Github pages needs to be enabled afterwards in the repo settings, select ``gh-pages`` branch, then re-run your workflow. 
+
+    The website we'll be at https://(user).github.io/(repo)/
+
+Publish your documentation with Sphinx and Read The Docs
+--------------------------------------------------------
+
+.. note:: Documentation to come!
+
+Document part of your package
+-----------------------------
+
+Sometimes, only a couple classes or modules are part of your public API, not all classes and modules need to be documented.  
+
+You can choose to document only a couple classes or modules passed with argument ``--html-subject``.
+
+.. warning:: The ``index.html`` won't be generated, you need to link to the specific HTML page. 
+
+Sphinx Integration
+------------------
+
+Sphinx object inventories can be used to create deep links between API
+documentation generated by pydoctor and by Sphinx.
+
+Linking to external API docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It can link to external API documentation using a Sphinx objects inventory
+with the following cumulative configuration option::
+
+    --intersphinx=http://sphinx-doc.org/objects.inv
+
+Linking to your API docs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+pydoctor's HTML generator will also generate a Sphinx objects inventory
+using the following mapping:
+
+* packages, modules -> py:mod:
+* classes -> py:class:
+* functions -> py:func:
+* methods -> py:meth:
+* attributes -> py:attr:
+
+To use this mapping in Sphinx, configure the `intersphinx extension`__::
+
+    intersphinx_mapping = {
+        'pydoctor': ('http://domain.tld/api', None),
+    }
+
+__ https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
+
+And use external references::
+
+    :py:func:`External API <pydoctor:pydoctor.model.Documentable.reparent>`
+
+    :py:mod:`pydoctor:pydoctor`
+    :py:mod:`pydoctor:pydoctor.model`
+    :py:func:`pydoctor:pydoctor.driver.getparser`
+    :py:class:`pydoctor:pydoctor.model.Documentable`
+    :py:meth:`pydoctor:pydoctor.model.Documentable.reparent`
+    :py:attr:`pydoctor:pydoctor.model.Documentable.kind`
+
+Customize builds
+----------------
+
+.. warning:: PyDoctor does not have a stable API yet. Custom builds are prone to break. Nevertheless, `PyDoctor's API documentation is available here <api/>`_. 
```

Comment:
This link is broken... I think that we can remove it and add it later in the Sphinx extension pluging

Reply:
Yes it was working with Github Pages, but for now I'll remove it